.github/workflows/.dco.yml

```diff
@@ -3,6 +3,15 @@ name: .dco
 
 # TODO: hide reusable workflow from the UI. Tracked in https://github.com/community/community/discussions/12025
 
+# Default to 'contents: read', which grants actions to # read commits.
```

Comment:
```suggestion
# Default to 'contents: read', which grants actions to read commits.
```

Think you missed a `#` here while formatting the comments (and on a lot of the other files as well?)

Reply:
doh 🙈 - fixed 👍